remote-processor/CMakeLists.txt

```diff
@@ -26,12 +26,19 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
+
 add_library(remote-processor SHARED
         Message.cpp
         RequestMessage.cpp
         AnswerMessage.cpp
         RemoteProcessorServer.cpp)
 
+include (GenerateExportHeader)
+generate_export_header(remote-processor
+                       BASE_NAME remote_processor)
```

Comment:
Yes, the file name is changed from remote-processor_export.h to remote_processor_export.h using this macro